src/modules/status.js

```diff
@@ -0,0 +1,20 @@
+import LocalStorage from './local_storage.js';
+
+const localS = new LocalStorage();
+
+class Status {
+    updateTaskCompletedStatus = (index, status) => {
+      const todo = localS.getLocalStorage();
+      todo[index].completed = status;
+      localStorage.setItem('todo', JSON.stringify(todo));
+    }
+
+    clearTaskCompleted = () => {
+      const todo = localS.getLocalStorage();
+
+      const newTodo = todo.filter((task) => task.completed === 0);
```

Comment:
<img width="896" alt="Screen Shot 2022-07-15 at 12 10 07" src="https://user-images.githubusercontent.com/82426682/179274026-314fa613-d848-4a35-8f78-3dce4b626662.png">

Your indexes are zero-based and the `completed` key has numeric values, not boolean values. Please fix both.